docs/developer-manual/terms-of-service.md

```diff
@@ -0,0 +1,157 @@
+---
+id: terms of service
+sidebar_position: 5
+slug: /tos
+---
+
+# TERMS OF SERVICE
+
+<!-- markdownlint-disable MD036 -->
+<!-- To be linked to from `"tos":` -->
+
+Valid from: *August 1, 2024* 
+
+<!-- markdownlint-restore -->
+
+These Terms of Service, together with any documents and additional terms they
+incorporate by reference (collectively, these “Terms”), are entered into between
+Orcfax Ltd. ("Orcfax”, “we,” “us,” and “our”), a legal entity registered as a
+corporation in the British Virgin Islands and you or the company or
+other legal entity that you represent (“you” or “your”).
+
+Please read these Terms carefully as they govern your use of the data that
+we (1) publish inline to transactions on the Cardano blockchain ("Oracle data"), 
+(2) data that we archive to the Arweave network ("Archival data") (3) or data that
+we publish to the pages and files located at the Internet domain Orcfax.io (the "Website").
+
+## THE SERVICES
+Collectively, the Oracle data collected, calculated, validated and published
+by Orcfax to transactions on the Cardano blockchain; along with the related Archival 
+data and files that Orcfax archives to the Arweave network; as well as the data and
+information published to the Website (including any and all sub-domain sites such as
+the Orcfax Explorer and Docs sites) are hereby referred to, collectively, as the 
+"Services" provided by Orcfax. 
+
+## REVISION OF TERMS OF SERVICE
+We reserve the right, in our sole discretion, to modify these Terms from time to
+time. If we make changes, we will provide you with notice of such changes using
```

Comment:
```suggestion
We reserve the right, in our sole discretion, to modify these Terms. If we make changes, we will provide you with notice of such changes using
```

Reply:
wondering if "we will provide you" is a bit folksy? -- suggest "notice will be provided"?

Reply:
the "we" and "you" language (as well as "from time to time" comes from the original disclaimer templates FYI but I agree with your preference to make the sentence cleaner and simpler and this rewording does not change or restrict the original intention.